msyncd/AccountsHelper.cpp

```diff
@@ -100,7 +100,6 @@ void AccountsHelper::slotAccountCreated(Accounts::AccountId id)
                 delete syncProfile;
             }
         }
-        delete newAccount;
     }
```

Comment:
It doesn't leak in an unbounded fashion (since the manager will only ever allocate the account with a given id once) but it does mean that every account will be instantiated once and remain in memory for the lifetime of the process.